src/test/util/matchable.rs

```diff
@@ -157,21 +157,9 @@ impl Matchable for Document {
             if k == "upsertedCount" {
                 continue;
             }
-            // TODO RUST-48: Remove this logic to bypass read concerns with an afterClusterTime
```

Comment:
Even though the relevant tests are no longer using the legacy runner, I removed these skips in case other tests happen to be affected by them.